src/model/functions.js

```diff
@@ -614,7 +614,7 @@ class Functions {
   }
 
   getSupervisorHost () {
-    return `http://${this.config.supervisorHost}:${this.config.supervisorPort}`;
+    return `http://localhost:${this.config.supervisorPort}`;
```

Comment:
This is used by code that runs inside the emulator service to talk to itself (lines 392 and 538 - `createFunction` and `deleteFunction`). Neither `host` nor `bindHost` are guaranteed to be routable from the emulator process. For example, `bindHost` could be `0.0.0.0` and `host` could be the IP of the other side of the NAT (the docker host for example).

I forgot about this detail in my original PR description, but this effectively eliminates the possibility of the three services (rest, grpc, supervisor) from running on different hosts. Since the consolidation of the `{grpc,rest,supervisor}Host` arguments already have this effect, I think this is OK..